### ISSUE TITLE
Our dependency on SFML should be hidden as much as possible.

### ISSUE THREAD

**Issue by smokyturbo** [grizzly-studios/Space-Evaders#60]
This would probably be a good thing to do for our next project.

We should make sure any SFML related code is hidden such that should we decide to use another framework such as Allegro/SDL etc we have to change as little code as possible.

**Comment by jademonkey** [comment]
Agreed. We should possibly create some sort of manager that acts as a wrapper around SFML code so we are calling the manager internally to the program and the manager calls the SFML. This means it would be easier to switch out SFML in the future as well. (Even switch it out for our own engine if we ever did that )
